current/local-cli/runWindows/utils/build.js

```diff
@@ -20,7 +21,11 @@ function buildSolution(slnFile, buildType, buildArch, verbose) {
 }
 
 function restoreNuGetPackages(options, slnFile, verbose) {
-  let nugetPath = options.nugetPath || path.join(options.root, 'node_modules/react-native-windows/local-cli/runWindows/.nuget/nuget.exe');
+  const nugetPath = options.nugetPath || `${process.env.TEMP}\\nuget.4.9.2.exe`;
```

Comment:
probably use : require('os').tmpdir() instead of process.env.TEMP.  It should be basically the same, but it'll align the correct ordering of env.TEMP, env.TMP  etc fallbacks